### DISCUSSION TITLE
500 Error Page lacks sufficient info to debug

### DISCUSSION THREAD

**Discussion by DeflateAwning** [open-webui/open-webui#2434] [General]
Per #2417, I am getting a 500 error. The issue was converted to a "discussion".

My real issue, which I believe should be fixed, is that the 500 error page indicates no useful information in trying to debug whatever the problem is. 

## Proposed Solutions:
* Option 1: Make the 500 error page show info about the cause of the error.
* Option 2: Add and document (in the README) a debugging environment variable to enable more detailed output on 500 error pages

**Comment by DeflateAwning** [comment]
Help me out here, @tjbck. Why is this converted to a discussion? There's a change that I think needs to be implented.

**Comment by haricalzi** [comment]
I have faced the same problem, read [here](https://github.com/open-webui/open-webui/discussions/2315#discussion-6677184) for a further explaination.
